storage/redis_controller_test.go

```diff
@@ -24,7 +27,7 @@ func TestRecoverLoop(t *testing.T) {
 
 	conf := config.Default
 
-	rc := NewRedisController(ctx)
+	rc := NewRedisController()
```

Comment:
So ctx argument is optional here?

Reply:
Not really, if doesn't event receives that value in this branch. In other hand, in master branch the signature for this method is:

`func NewRedisController(ctx context.Context) *RedisController`

So, it wasn't even running 😄 . I didn't change the function `NewRedisController` in order to not make big changes